boards/arm/olimexino_stm32/board.cmake

```diff
@@ -1,6 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
-board_runner_args(jlink "--device=STM32F103RB" "--speed=4000")
+board_runner_args(stm32flash "--baud-rate=115200" "--start-addr=0x08000000")
```

Comment:
The runners are not mutually exclusive. Why not just default to stm32flash and keep all the others instead? The first include you do is the default.

Reply:
Done

Reply:
You've removed the JLink configuration line; it should still be there.  The arguments to the board runners are orthogonal.